pysat/tests/test_utils.py

```diff
@@ -56,10 +56,58 @@ def test_deprecation_warning_computational_form():
     assert war[0].category == DeprecationWarning
 
 
+class TestDeprecation():
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        warnings.simplefilter("always", DeprecationWarning)
+        # store current pysat directory
+        self.data_path = pysat.data_dir
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+        pysat.utils.set_data_dir(self.data_path)
+
+    def test_set_data_dir_deprecation(self):
+        """Test for deprecation warning when setting data_dir path"""
+
+        wmsg = 'In 3.0.0 pysat will move to a central location for storing'
+
+        with warnings.catch_warnings(record=True) as war:
+            pysat.utils.set_data_dir('.')
+
+        found_msg = False
+        for iwar in war:
+            print(iwar.message)
+            if iwar.category == DeprecationWarning \
+                    and str(iwar.message).find(wmsg) >= 0:
+                found_msg = True
+
+        assert found_msg, "didn't find warning about: {:}".format(wmsg)
```

Comment:
Recommend using warning evaluation function

Reply:
Happy to do that but I don't know what you mean. I grabbed this code from somewhere else in pysat. I'll look into it.

Reply:
The function in #755?

Reply:
Yup!  There are a few examples in the unit tests.  Now that you have the function name, I'm sure it's easy enough to find.